server/tests/server/planner/test_stages.py

```diff
@@ -220,8 +207,7 @@ def test_storagerescan_largedataset(runner, queue_factory, host_factory):  # pyl
     queue = queue_factory.create(name='queue1', config=yaml_dump({'module': 'nmap', 'args': 'arg1'}))
     existing_targets_count = 10**6
     existings_targets_vals = [
-        str((queue.id, str(ip_address(idx)), SchedulerService.hashval(str(ip_address(idx)))))
-        for idx in range(existing_targets_count)
+        str((queue.id, str(ip_address(idx)), SchedulerService.hashval(str(ip_address(idx))))) for idx in range(existing_targets_count)
```

Comment:
remove from commit, re-formated for no reason

and drop all other unnecessary formatting from this PR as well please